src/webextension/locales/en-US/firstrun.ftl

```diff
@@ -9,10 +9,15 @@ welcome-intro =
     new entries, and then later you can view, search, edit, and
     delete those entries.
 
+welcome-warning =
+    This is a prototype that is rapidly evolving with many great improvements.
+    Much of the functionality seen in this Alpha is expected to change. As such,
+    data stored here may not be retained in future updates.
```

Comment:
I sort of feel like this is burying the lede... while it's nice that it's in bold (so people are likely to read this first), the actual *important* bit of this text is the third sentence, and is a little bit indirect. If we're not guaranteeing future retention of users' data (and we think people would expect us to), then I think we should be more direct about it. Something like the following would make me feel more comfortable:

> Don't rely solely on this extension for storing your passwords! This is a prototype which is rapidly evolving, and as such we can't guarantee that data will be retained in future updates.

Reply:
I think this suggested language is too scary.  If there's concern the current text is not explicit enough, my suggestion would be:

> This is a prototype that is rapidly evolving with many great improvements.
> Much of the functionality seen in this Alpha is expected to change. As such,
> data stored here is not guaranteed to be retained in future updates.

If there's concern the text is also possibly too verbose, might I suggest this:

> This is a rapidly evolving prototype that will change radically as Alpha progresses.
> Any data stored here is not guaranteed to be retained in future updates.

Reply:
Good input, thanks, you two.

Ultimately this is @sandysage's call, especially as she's been consulting with other trusted folks on this as well...

Reply:
Given these points, let's trim the message to be:

> This is a rapidly evolving prototype that will change as Alpha progresses. Any data stored here is not guaranteed to be retained in future updates.

Which only states the warning: doesn't bury the lede, but also doesn't over-emphasize the risk involved.